src/Aspire.Hosting.Keycloak/Aspire.Hosting.Keycloak.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>aspire hosting keycloak</PackageTags>
+    <Description>Keycloak support for .NET Aspire.</Description>
```

Comment:
Is there an icon/logo we can use for these new packages?

Reply:
There is, and it was in this PR already, but it seems like we can't just add one without permission from Keycloak folks, so I removed it.

Reply:
Can we create an issue to track this and link to an issue opened on the Keycloak repo asking for permission?

Reply:
> Can we create an issue to track this and link to an issue opened on the Keycloak repo asking for permission?

Sure, here are the issues:
On Keycloak: https://github.com/keycloak/keycloak/issues/31215
Ours: https://github.com/dotnet/aspire/issues/4856